app/models/type.rb

```diff
@@ -72,9 +72,9 @@ def <=>(type)
     name <=> type.name
   end
 
-  def self.all
-    find(:all, order: 'position')
-  end
+  # def self.all
+  #  find(:all, order: 'position')
+  # end
```

Comment:
I think we are not understanding eachother here, let me try with my less polite voice:

***Delete that commented code already!***

;-)

Best regards :cat: